doc/content/reference/api/field-mask.md

```diff
@@ -6,17 +6,185 @@ weight: -1
 
 ## Field Masks
 
-{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a reading request, or to specify fields that should be updated in a writing request. See Google's [Protocol Buffers reference](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.FieldMask) for more information about field masks.
+{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a reading request, or to specify fields that should be updated in a writing request. By default, API requests will not return or update most fields, unless they are specified in a field mask.
```

Comment:
It depends on the request. Not all requests take field masks.

Reply:
Updated, it looks like all requests which operate this way list a `field_mask` parameter, i.e https://www.thethingsindustries.com/docs/reference/api/application/#message:UpdateApplicationRequest or http://localhost:1313/reference/api/end_device/#message:GetEndDeviceRequest